libs/sdk-ui-charts/src/highcharts/chartTypes/waterfallChart/tests/waterfallChartOptions.test.ts

```diff
@@ -0,0 +1,204 @@
+// (C) 2023 GoodData Corporation
+import { ReferenceRecordings } from "@gooddata/reference-workspace";
+import { DefaultColorPalette, IColorAssignment, localIdentifierMatch } from "@gooddata/sdk-ui";
+import { ScenarioRecording } from "@gooddata/sdk-backend-mockingbird";
+import { IColorDescriptor } from "@gooddata/sdk-model";
+
+import { recordedDataFacade } from "../../../../../__mocks__/recordings";
+import { WaterfallChartColorStrategy } from "../waterfallChartColoring";
+import { getWaterfallChartSeries } from "../waterfallChartsSeries";
+import { getMVS } from "../../_util/test/helper";
+import { buildWaterfallChartSeries, getColorAssignment, getTotalColumnColor } from "../waterfallChartOptions";
+import { IChartConfig, ITotalConfig } from "../../../../interfaces";
+import { ISeriesItem } from "../../../typings/unsafe";
+import { getDrillableSeries } from "../../_chartOptions/chartDrilling";
+
+const emptyHeaderTitle = "Total";
+
+describe("waterfallChartOptions:", () => {
```

Comment:
There is no reason to add colon to describe blocks. IMHO we do not do that anywhere else. Please, remove it from all the tests you added.

Reply:
I have removed the colon from all my describe blocks. Thank you a lot!